### PR TITLE
Make 'System language' translatable, add bottom margins to settings scroll views

### PR DESCRIPTION
**Pull request by nirvn**
A couple of follow up tweaks to the custom language feature.

@suricactus , I've tweaked things so we are adding a translatable system language item to the list of supported languages vs. embedding it in, it'll make it easier to switch to a dynamically generated list of supported languages down the road.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d70124d6ac3e9b1777950f0d49b2be9109770e9c:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-f7c04412b3f2d3d7c296e419eef8793701298fb0-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-f7c04412b3f2d3d7c296e419eef8793701298fb0-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-f7c04412b3f2d3d7c296e419eef8793701298fb0-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-f7c04412b3f2d3d7c296e419eef8793701298fb0-x86.apk)

File: src/qml/QFieldSettings.qml [hunk 1]
```diff
@@ -298,16 +298,18 @@ Page {
...
                           }
 
                           Component.onCompleted: {
                               var language = settings.value('customLanguage', '');
-                              var keys = Object.keys(languages);
-                              model = Object.values(languages);
+                              var keys = [""].concat(Object.keys(languages));
+                              var items = [qsTr( "System language" ) + (qsTr( "System language" ) !== "System language" ? ' (System language)' : '')]
+                              model = items.concat(Object.values(languages));
+
```
Comment (suricactus) [inline src/qml/QFieldSettings.qml:312]:
```suggestion
                              var keys = [''].concat(Object.keys(languages));
                              var systemLangTranslation = qsTr('System language');
                              var systemLangSuffix = systemLangTranslation !== 'System language' ? ' (System language)' : ''
                              var items = [systemLangTranslation + systemLangSuffix]
                              model = items.concat(Object.values(languages));

```

File: src/qml/QFieldSettings.qml [hunk 2]
```diff
@@ -278,8 +279,7 @@ Page {
                           Layout.fillWidth: true
                           Layout.alignment: Qt.AlignVCenter
 
-                          property variant languages: {"":"System language",
-                              "en":"English",
+                          property variant languages: {"en":"English",
```
Comment (suricactus) [inline src/qml/QFieldSettings.qml:282]:
```suggestion
                          property variant languages: {
                              "en":"English",
```

**Review by suricactus — APPROVED** [review]
Nice. Small code style comments, merge when ready.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9b87fb9478473ead4c58ce53a3ae57ba0ddca507:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-434dd95269fb70cbb46da338126819b3adfcc486-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-434dd95269fb70cbb46da338126819b3adfcc486-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-434dd95269fb70cbb46da338126819b3adfcc486-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2188-434dd95269fb70cbb46da338126819b3adfcc486-x86.apk)

**Pull request merged**